databases/relational-databases.md

```diff
@@ -161,12 +262,39 @@ Schema design:
 Further learning: Object-Relational Mapping (& ORM libaries)
 - [freeCodeCamp: Which JavaScript ORM should you be using in 2018?](https://medium.freecodecamp.org/a-comparison-of-the-top-orms-for-2018-19c4feeaa5f)
 
-### Check for Understanding
-
+Solve the quizzes mentioned below
 - [W3Schools quiz](http://www.w3schools.com/sql/sql_quiz.asp)
-- [JavaTPoint sql quiz](https://www.javatpoint.com/sql-quiz)
+- [JavaTPoint SQL quiz](https://www.javatpoint.com/sql-quiz)
 - [TutorialsPoint quiz](https://www.tutorialspoint.com/sql/sql_online_quiz.htm)
 
+### Check for Understanding
+
+- What do you mean by Relational database?
+- What is CRUD in SQL?
+- What is DDL?
+- How does SELECT command work?
+- How does JOIN work?
+- What are the advantages of a Relational database?
+- What is an ORM?
+- How is PostgreSQL different from MySQL?
+- How do you model data for relational databases?
+
+
+### Supplemental Resources
+
+- [Medium: Omar El Gabry's Tutorials on Databases, Modeling, Design Process, & Normalization](https://medium.com/omarelgabrys-blog/database-introduction-part-1-4844fada1fb0)
+- [TutorialsPoint, detailed](https://www.tutorialspoint.com/sql/)
+- [Codecademy Tutorial, interactive](https://www.codecademy.com/learn/learn-sql)
+- [Learn SQL the hard way, textbook](https://learncodethehardway.org/sql/)
+- [GalaXQL, interactive, graphical](http://sol.gfxile.net/galaxql.html)
+- [PostgreSQL SQL, technical reference manual](https://www.postgresql.org/docs/current/static/sql.html)
+- [AgileData: Data Modeling 101](http://www.agiledata.org/essays/dataModeling101.html)
+- [LucidChart: What is a Database Model?](https://www.lucidchart.com/pages/database-diagram/database-models)
```

Comment:
Double check these links. its not supplemental if you are assigning it in the outline.